samples/net/sockets/echo_client/prj.conf

```diff
@@ -2,13 +2,16 @@
 CONFIG_NETWORKING=y
 CONFIG_NET_UDP=y
 CONFIG_NET_TCP=y
+#CONFIG_NET_TCP2=y
```

Comment:
Please create an overlay config file `overlay-tcp2.conf` for these settings.

Reply:
Done.